src/rpc/mining.cpp

```diff
@@ -1035,6 +1140,7 @@ static const CRPCCommand commands[] =
     { "mining",             "prioritisetransaction",  &prioritisetransaction,  {"txid","dummy","fee_delta"} },
     { "mining",             "getblocktemplate",       &getblocktemplate,       {"template_request"} },
     { "mining",             "submitblock",            &submitblock,            {"hexdata","dummy"} },
+    { "mining",             "pprpcsb",                &pprpcsb,            {"header_hash","mix_hash", "nonce"} },
```

Comment:
Little bit nit picky here but, space after the comma. You can fix it now, or forget about it when we do a code cleanup round.